dali/python/nvidia/dali/data_node.py

```diff
@@ -70,31 +75,37 @@ def gpu(self):
 
     def __add__(self, other):
         return _arithm_op("add", self, other)
+
```

Comment:
This defeats the logical connection between these functions. PEP8 says it's acceptable to have no blank lines between one-liners, so maybe these function should be one-liners?

Reply:
I get that __op__ and __rop__ are pairs, but don't really see that huge difference with this one `\n` or not as long as those are next to each other. But that's me.

Reply:
Double new line between methods is a Python standard, so I would leave it as the tool did it.